cristin-biobank/src/main/java/no/unit/nva/biobank/common/MappingUtils.java

```diff
@@ -0,0 +1,113 @@
+package no.unit.nva.biobank.common;
+
+import no.unit.nva.biobank.model.cristin.CristinAssocProjectForBiobank;
+import no.unit.nva.biobank.model.cristin.CristinBiobank;
+import no.unit.nva.biobank.model.cristin.CristinBiobankMaterial;
+import no.unit.nva.biobank.model.cristin.CristinCoordinator;
+import no.unit.nva.biobank.model.nva.Biobank;
+import no.unit.nva.biobank.model.nva.BiobankApprovals;
+import no.unit.nva.biobank.model.nva.BiobankMaterial;
+import no.unit.nva.biobank.model.nva.ExternalSourcesBiobank;
+import no.unit.nva.biobank.model.nva.TimeStampFromSource;
+import no.unit.nva.cristin.model.CristinInstitution;
+import no.unit.nva.cristin.model.CristinOrganization;
+import no.unit.nva.cristin.model.CristinUnit;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public final class MappingUtils {
+    private static final String INSTITUTIONS_PATH_ELEMENT = "institutions";
+    private static final String UNITS_PATH_ELEMENT = "units";
+    private static final String PERSONS_PATH_ELEMENT = "persons";
+
+    private MappingUtils() {
+        // no-op
+    }
+
+
+    /**
+     * The new NVA biobank from Cristin Biobank.
+     *
+     * @param cristinBiobank - cristin model
+     * @param domainName     - env
+     * @param basePath       - path
+     */
+    public static Biobank cristinModelToNvaModel(CristinBiobank cristinBiobank,
+                                                 String domainName,
+                                                 String basePath
+    ) {
+        var id = getBiobankUri(cristinBiobank.getBiobankId(), domainName, basePath);
```

Comment:
You have some typos here I think. Is it not supposed to be `coordinatingInstitution`? Same applies to variable and getter in `CristinBiobank`.